packages/components/src/KaizenProvider/KaizenProvider.tsx

```diff
@@ -1,21 +1,21 @@
 import React from "react"
-import { Theme as BaseTheme } from "@kaizen/design-tokens"
-import { OptionalIntlProvider } from "./OptionalIntlProvider"
-import { ThemeProvider } from "./ThemeProvider"
+import { FontDefinitions } from "./subcomponents/FontDefinitions"
+import { OptionalIntlProvider } from "./subcomponents/OptionalIntlProvider"
 
-export type KaizenProviderProps<Theme extends BaseTheme = BaseTheme> = {
+export type KaizenProviderProps = {
```

Comment:
So, breaking change? Is it just fixing the next-template?

Reply:
Going to sneak this in, cause no one uses the `theme` prop on `KaizenProvider` 😄

Reply:
🥷 🙈